### PR TITLE
Milestone 4: calculations

### PR DESCRIPTION
**Pull request by aliciapaz**
### Changes to the folder structure
- Added 'logic' folder inside 'src' directory
- Initialized 'calculate' and 'operate' files inside it

### Code
- Implemented `calculate` and `operate` as arrow functions
- Used `big-js` library to handle numbers
- Imported `calculate` into `App`

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+// this is commented out to avoid the linter error 'is defined but never used'
+// import calculate from '../logic/calculate';
```
Comment (rindrajosia) [inline src/components/App.js]: [REQUIRED] calculate.js should be imported in the App component

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- calculate.js module is added :+1:
- operate.js module is added :+1:

To improve :factory_worker: [REQUIRED]:

- [x] Please, make sure that there is no error in your readme because it the first thing the user checks (For example, the title of your Project)
- [x] `calculate.js` should be imported in the App component


- [x] Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.
- [x] It will be nice if you use functions provided by `Big.js` like `multiply, minus, plus, ...`
- [x]  Kindly implement for other corresponding buttonName in your `calculate.js` file
- [x] calculate accepts calculator `data object` and a `button name` as arguments. The 
`data object` contains three properties: `total`, `next` and `operation` 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/totalNextOperate.png))

Nice to have [OPTIONAL] :

- [ ] 

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

File: src/components/App.js (hunk 1 above)
Comment (aliciapaz) [reply src/components/App.js]: I know but if I leave it there will be a linter error and the checks will not pass.
Should I disable Eslint on that line?

File: src/components/App.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate'; // eslint-disable-line no-unused-vars
```
Comment (enaburekhan) [inline src/components/App.js:4]: - [x] Remove the `eslint-disable-line no-unused-vars`. And place <calculate /> inside the react.fragment short syntax.

**Review by enaburekhan — CHANGES_REQUESTED** [review]
# Status: Changes required::full_moon:wrench::hammer:
**Hi Alicia!
Great job so far:clap::clap:. However, we just need to make a few changes to get this project approved:muscle:**

**To improve**

- Correct the readme title from `CLCULATOR` to `CALCULATOR`.
- Check my inline comment and make the required changes.

Request another review when you are ready.

Do have questions/comments? kindly drop them :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy hacking:blush:
Eric Enaburekhan.

**Comment by aliciapaz** [comment]
### Changes implemented
- Update readme
- Since 'calculate' is not a React component, it can't be rendered as it was one (between the React.fragment shorthand as suggested). Therefore, I disabled Eslint for a single line to avoid linter error due to importing a module that is not used (yet)

**Review by taiwo2 — APPROVED** [review]
# STATUS: APPROVED 🎉 
@aliciapaz  well done for implementing the suggested changes. kindly merge and prepare for the next task. Thanks

**Pull request merged**